_data/meltano/extractors/tap-google-analytics/meltanolabs.yml

```diff
@@ -192,20 +210,11 @@ settings:
     ![Screenshot of open account selector](/assets/images/tap-google-analytics/account-selector-open.png)
 
     4. You will see the View ID displayed inside the selector below the name of the view (e.g. "All Web Site Data"): `188274549`
+  kind: password
   label: View ID
   name: view_id
   placeholder: Ex. 198343027
 settings_group_validation:
-- - key_file_location
```

Comment:
This is tricky, I need to figure out how to handle this. The current tap settings dont explicitly define the combinations that are allowed.